<ui version="4.0" >
 <class>customStatusDialogClass</class>
 <widget class="QDialog" name="customStatusDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>193</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Custom status</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../../qutim.qrc" >
    <normaloff>:/icons/crystal_project/statuses.png</normaloff>:/icons/crystal_project/statuses.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>4</number>
   </property>
   <property name="spacing" >
    <number>3</number>
   </property>
   <item row="0" column="0" colspan="2" >
    <widget class="QLineEdit" name="captionEdit" >
     <property name="maxLength" >
      <number>30</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QTextEdit" name="awayEdit" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy" >
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy" >
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QListWidget" name="iconList" >
     <property name="showDropIndicator" stdset="0" >
      <bool>false</bool>
     </property>
     <property name="dragDropMode" >
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="movement" >
      <enum>QListView::Static</enum>
     </property>
     <property name="spacing" >
      <number>5</number>
     </property>
     <property name="viewMode" >
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QPushButton" name="chooseButton" >
     <property name="text" >
      <string>Choose</string>
     </property>
     <property name="icon" >
      <iconset resource="../../../qutim.qrc" >
       <normaloff>:/icons/crystal_project/apply.png</normaloff>:/icons/crystal_project/apply.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QPushButton" name="cancelButton" >
     <property name="text" >
      <string>Cancel</string>
     </property>
     <property name="icon" >
      <iconset resource="../../../qutim.qrc" >
       <normaloff>:/icons/crystal_project/cancel.png</normaloff>:/icons/crystal_project/cancel.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="../../../qutim.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>customStatusDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>170</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>194</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
